images/master/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM google/debian:wheezy
```

Comment:
I think you just need a few build flags to statically link stuff, then you could use FROM scratch.